### PR TITLE
Regenerate SDK for Kafka REST API

### PR DESCRIPTION
**Pull request by linouk23**

### CODE REVIEW

File: kafkarest/v3/api_cluster_v3.go [hunk 1]
```diff
@@ -42,59 +42,58 @@ var (
...
+			 @return ApiGetKafkaClusterRequest
 	*/
-	GetKafkaV3Cluster(ctx _context.Context, clusterId string) ApiGetKafkaV3ClusterRequest
+	GetKafkaCluster(ctx _context.Context, clusterId string) ApiGetKafkaClusterRequest
 
-	// GetKafkaV3ClusterExecute executes the request
+	// GetKafkaClusterExecute executes the request
 	//  @return ClusterData
-	GetKafkaV3ClusterExecute(r ApiGetKafkaV3ClusterRequest) (ClusterData, *_nethttp.Response, error)
+	GetKafkaClusterExecute(r ApiGetKafkaClusterRequest) (ClusterData, *_nethttp.Response, error)
```
Comment (mtodzo) [inline kafkarest/v3/api_cluster_v3.go:59]: Are you ok with losing the V3 part in the function names? Doesn't matter to me

Comment (linouk23) [reply kafkarest/v3/api_cluster_v3.go:59]: That's a great catch! I think the team responsible for this API made this decision on purpose actually.

**Review by mtodzo — APPROVED** [review]
lgtm - just the one question about naming changes

File: kafkarest/v3/api_cluster_v3.go (hunk 1 above)
Comment (mtodzo) [reply kafkarest/v3/api_cluster_v3.go:59]: All of the APIs seem to have that same change

**Pull request merged**